langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -69,6 +93,11 @@ public isolated function get(map<Type> m, string k) returns Type = @java:Method
 
 # Returns a map containing [key, member] pair as the value for each key.
 #
+# ```ballerina
+# map<int> indices = {a: 1, b: 2, c: 3, d: 4};
+# map<[string, int]> indexEntries = indices.entries();
```

Comment:
Thinking out loud, can index/indices be a bit confusing if interpreted to be related to some index of the map itself (e.g., keys). Can we use a different map?